### PR TITLE
My solution for CodeForces problem #548

### PR DESCRIPTION
**Pull request by YifatYank**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            line = input()
+            line = line.split(' ')
+            cell_type = line[0]
+            value = int(line[1])
+            cell = CellInfo(i, value, cell_type)
+            self._cell_array.append(cell)
+
+    # Fills the class's cell_array with data from a file. The function gets the file's name as a parameter.
+    def get_map_from_file(self, file_name):
+        file = open(file_name)
```
Comment (idlv75) [inline main.py:85]: The Input should be in yaml format.
You are handling it as text file.
Can you implement the same with the standard Python yaml library?

**Review by idlv75 — CHANGES_REQUESTED** [review]
Please see comments inline

File: main.py (hunk 1 above)
Comment (YifatYank) [reply main.py:85]: yes, I will

**Comment by idlv75** [comment]
Invited for interview

**Pull request closed without merging**